riscv/triggers.cc

```diff
@@ -155,7 +155,7 @@ void mcontrol_t::tdata1_write(processor_t * const proc, const reg_t val, const b
     case MATCH_LT:
     case MATCH_MASK_LOW:
     case MATCH_MASK_HIGH:
-      match = (triggers::mcontrol_t::match_t) match_value;
+      match = (match_t) match_value;
```

Comment:
In 05de1277500b770a9fb2f83896f00b596f4b51d0 I don't know what "indicator of variables" means...how about "remove redundant namespace qualifiers"